alt1/scripts/notifications.ts

```diff
@@ -150,14 +161,42 @@ export function notifyEvent(event: EventRecord): void {
     }
 }
 
-export function setDefaultTitleBar() {
-    alt1.setTitleBarText(`<span title='${getStockTitle()}'>Listening for DSF events...</span>`);
+export function registerStatusUpdates() {
+    if (STATUS_API_URL) {
+        const settings = {
+            favoriteEvents: localStorage.getItem("favoriteEvents") ?? null,
+            notificationModes: localStorage.getItem("notificationModes") ?? null,
+        };
+        alt1.registerStatusDaemon(`${STATUS_API_URL}/status`, JSON.stringify({settings}));
+    }
+}
+
+export function updateTitlebar() {
+    const eventInProgress = JSON.parse(localStorage.getItem("eventInProgress") ?? '') as EventInProgress;
```

Comment:
```suggestion
    const eventInProgress = JSON.parse(JSON.parse(localStorage.getItem("eventInProgress")) ?? '') as EventInProgress;
```
For the `setDefaultTitleBar`, the value is saved as "null", so loading without parsing will return "null" ?? "" -> "null"